ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/components/SignatureCountLabelLayout.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { Button, Flex, FlexItem, Label, Popover, Text } from '@patternfly/react-core';
+
+import useMetadata from 'hooks/useMetadata';
+import { getVersionedDocs } from 'utils/versioning';
+import ExternalLink from 'Components/PatternFly/IconText/ExternalLink';
+import PopoverBodyContent from 'Components/PopoverBodyContent';
+
+export type SignatureCountLabelProps = {
+    count: number;
+};
+
+const noSignatureMessage = 'No signature found';
+
+function getBodyContent(version: string) {
+    return (
+        <Flex direction={{ default: 'column' }}>
+            <FlexItem>
+                <Text>
+                    Image signatures increase the security and transparency of container images.
+                </Text>
+            </FlexItem>
+            <FlexItem>
+                <Text>
+                    Create at least one image signature integration to download and verify image
+                    signatures.
+                </Text>
+            </FlexItem>
+            <FlexItem>
+                <Text>
+                    For more information, see{' '}
+                    <ExternalLink>
+                        <a
+                            href={getVersionedDocs(version, 'operating/verify-image-signatures')}
+                            target="_blank"
+                            rel="noopener noreferrer"
+                        >
+                            RHACS documentation
```

Comment:
technically we should be using the proper branding name here -- see `productBranding.ts`